src/lando/api/tests/test_transplants.py

```diff
@@ -1824,3 +1863,64 @@ def test_warning_multiple_authors(phabdouble, mocked_repo_config, create_state):
     assert (
         warning.details == "Revision has multiple authors: alice, bob."
     ), "Multiple authors on a revision should return a warning."
+
+
+@pytest.mark.django_db(transaction=True)
+def test_transplant_on_linked_legacy_repo(
```

Comment:
I think this was probably accidentally removed in the merge, and re-added here. It should be removed from this PR and rolled up in the merge commit (this file was one that had conflicts).

Reply:
This wasn't removed in the merge PR, the merge PR pre-dates this change. This change was added during the work week, so I had to merge changes landed after I started the initial merge back into this branch. I can't remove it from the merge commit because it doesn't exist there yet.

Reply:
>This wasn't removed in the merge PR, the merge PR pre-dates this change

This is no longer true as of yesterday, after you force pushed as seen [above](https://github.com/mozilla-conduit/lando/pull/224#event-16462467560) and reverted some files to main.

Everything added in the work week should already be part of the "merge PR" (i.e., [api-merge-without-fixes](https://github.com/mozilla-conduit/lando/tree/api-merge-without-fixes), #225). This PR (#224) should only have the fixes. I think this is the same case (and fix) as what you did with the other files you already fixed in the merge commit in #225 yesterday (e.g., `src/lando/api/legacy/workers/base.py`).

Reply:
This still shows up here? This indicates there is something wrong in the merge PR. This test is already in `main`.